### PR TITLE
Expose the reuse_fips flag on os_server

### PR DESCRIPTION
**Pull request by emonty**

### CODE REVIEW

**Comment by gregdek** [comment]
Thanks again to @emonty. This module is going into community review.

We notice that you are one of the maintainers (@emonty, @shrews, @juliakreger, @j2sol, @rcarrillocruz) of this module, so when you think it's ready, please comment "shipit" and we will consider it for merging.

[This message brought to you by your friendly Ansibull-bot.]

**Review by Shrews — APPROVED** [review]
LGTM

File: cloud/openstack/os_server.py [hunk 1]
```diff
@@ -428,6 +440,7 @@ def _create_server(module, cloud):
     flavor = module.params['flavor']
     flavor_ram = module.params['flavor_ram']
     flavor_include = module.params['flavor_include']
+    reuse_fips = module.params['reuse_fips']
```
Comment (omgjlk) [inline cloud/openstack/os_server.py]: I don't see where this is actually used. Later in this function you directly access `module.params['reuse_fips']`.

Am I missing something?

**Review by omgjlk — COMMENTED** [review]
I have a question...

**Review by Shrews — COMMENTED** [review]
Ok, _NOW_ it LGTM

**Review by Shrews — CHANGES_REQUESTED** [review]
ignore this

**Review by Shrews — APPROVED** [review]

**Comment by bryfry** [comment]
which version of shade actually uses this? I'm getting unexpected keyword argument 'reuse_fips' now.

**Comment by Jorge-Rodriguez** [comment]
@Shrews I tried this with shade 1.21.1 (latest) and got the same result as @bryfry 

Moreover, there seems to be no functional code in the shade codebase with the names `reuse_fips` or `fips`. This PR introduces a breaking change to the module.

@emonty care to weigh in on this?

**Comment by Shrews** [comment]
Yep, this is a bug. Correct parameter name is reuse_ips. Fix coming shortly.

**Comment by Shrews** [comment]
https://github.com/ansible/ansible-modules-core/pull/4939

**Pull request merged**